src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -563,6 +563,39 @@ public void AnalyzeCommand_DefaultEndToEndAnalysis()
             configurationNotificationCount.Should().Be(0);
         }
 
+        [Fact]
+        public void AnalyzeCommand_CachesNotificationsWhenComputingTargetHashesWithoutPersistingToLogFile()
+        {
+            try
+            {
+                ExceptionRaisingRule.s_exceptionCondition = ExceptionCondition.LoadingPdb;
+
+                // TODO: even though this test raises an exception per binary, we are not persisting
+                // the results to a log file. We therefore validate against the console output. 
+                // The MSBuildConverter we use for this converts all console notifications to results.
+                RunTests(persistLogFileToDisk: false, raiseNotificationPerBinary: false, expectedReturnCode: 1);
```

Comment:
>1 [](start = 109, length = 1)

`AnalyzeCommandBase.FAILURE` (and `SUCCESS` below)?

Reply:
Haha! Yes, of course. :)

---
In reply to: [316893217](https://github.com/microsoft/sarif-sdk/pull/1658#discussion_r316893217) [](ancestors = 316893217)